client-v2/src/components/App.tsx

```diff
@@ -101,6 +101,10 @@ const BackgroundHeader = styled('div')`
   width: 100%;
 `;
 
+export const frontsEditPath = `/:priority(${Object.keys(priorities).join(
```

Comment:
We need to derive priority from this route elsewhere, so it lives here and is exported.